Nav.php

```diff
@@ -164,6 +174,7 @@ public function renderItem($item)
         $items = ArrayHelper::getValue($item, 'items');
         $url = ArrayHelper::getValue($item, 'url', '#');
         $linkOptions = ArrayHelper::getValue($item, 'linkOptions', []);
+        $asDropdown = isset($item['inline']) ? !$item['inline'] : $this->asDropdown;
```

Comment:
Diffrent names 'inline' and 'asDropdown' will confuse developer. Single one should be used both for default optnion property and item option key.